forgerock-auth/src/main/java/org/forgerock/android/auth/callback/DeviceBindingCallback.kt

```diff
@@ -224,6 +230,11 @@ open class DeviceBindingCallback : AbstractCallback, Binding {
             return
         }
 
+        if (!deviceAuthenticator.validateCustomClaims(customClaims)) {
```

Comment:
deviceAuthenticator.validateCustomClaims(customClaims).not

Reply:
changed